recipes/taxator-tk/meta.yaml

```diff
@@ -0,0 +1,33 @@
+{% set name = "taxator-tk" %}
+{% set version = "1.3.3e" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://research.bifo.helmholtz-hzi.de/webapps/wa-download/download.php?id=v4Xdsvn6ib6AYQAyXmcTz6
+  md5: 6575138bb60503e9fdb76ff463be9df3
+
+build:
+  number: 1
+  skip: True # [osx]
+
+requirements:
+  build: 
+    - {{ compiler('c') }}
```

Comment:
nothing is being compiled so this is not necessary.
but do consider compiling it on the recipe if possible, its always better that way.